src/libstd/sys/windows/process.rs

```diff
@@ -141,12 +138,34 @@ impl Command {
                                    .with_extension(env::consts::EXE_EXTENSION);
                     if fs::metadata(&path).is_ok() {
                         return Some(path.into_os_string())
+                    } else {
```

Comment:
We typically try to avoid rightward drift where possible, so because of the `return` above you can omit this `else`, de-indenting what's below. You can also change what's below to:

```rust
let exts = match env_pathext {
    Some(e) => e,
    None => continue,
};
// ...
```

to get rid of another layer of indentation.